### ISSUE TITLE
New room list: Stop supporting old room list ordering options

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#14373]
Deferred to some time in the distant future when it's safe to remove the fallback

**Comment by gsouquet** [comment]
@turt2live I'm looking at this issue today, could you add some historical context so that I am able to complete this task?

**Comment by t3chguy** [comment]
I believe this is referring to the secret legacy custom ordering mode - https://github.com/matrix-org/matrix-react-sdk/blob/bdc56be8632bdd354ca48a4d06ef72b0937d1aac/src/stores/room-list/algorithms/tag-sorting/index.ts#L29

**Comment by gsouquet** [comment]
This issue is marked as `X-Blocked`, it would be great to add context on what it is blocked on and how we can action it.

**Comment by turt2live** [comment]
This is referring to a settings store migration that may or may not still exist. The custom ordering being secret is deliberate and not planned for removal (in this issue).

In theory the issue URL should appear in code somewhere to confirm, but if not I can dig up more history of needed.
